tests/integration/schema/default_fields.go

```diff
@@ -149,6 +150,14 @@ var aggregateFields = fields{
 	},
 }
 
+var similarityField = Field{
+	"name": "_similarity",
+	"type": map[string]any{
+		"kind": "SCALAR",
+		"name": "Float",
+	},
+}
+
```

Comment:
> todo: Please make sure you add an introspection test(s) to make sure the new input type shows up in the correct shape in the correct place without breaking the GQL spec.

@AndrewSisley I added this here to ensure it's part of introspection test. Some test were actually failing because it was missing. But otherwise I don't know how I could meaningfully test it further. If you have any suggestions I'm happy to add them.

Reply:
I actually forgot about the field, I am more concerned about it's params.

You need to test that the InputArgs are properly formed, and only present on fields of the relevant types.

The test directory is a bit of a mess, but `TestFilterForSimpleSchema` is an example of this for filters, `TestSchemaAggregateSimpleCreatesUsersAverage` does the same for `_avg`.

Reply:
Let me know what you think of the added introspection tests.

Reply:
They look good, thanks Fred :)